app/app.iml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
imho yes, since the iml is intellij's project file which gets generated automatically during project import.